requirements.txt

```diff
@@ -12,4 +12,5 @@ sphinx-autodoc-typehints
 sphinx_rtd_theme
 jinja2
 docutils
-
+sparse==0.9.1
```

Comment:
Do we need to pin `sparse` exactly or can we use a >= dependency?

Reply:
I could remove the pin but this issue cropped up: https://github.com/theislab/diffxpy/issues/212